artifacts/4PS/Invoke-4PSArtifactHandling.ps1

```diff
@@ -208,6 +208,15 @@ function Invoke-4PSArtifactHandling {
                                     -MethodName InitContainer `
                                     -Argument "$($env:AZP_SERVICE_DISPLAYNAME)"
                             }
+                            $ConstructBaseDETestInstalled = Get-NAVAppInfo -Id '39ea957f-9416-49f4-8b0a-08ba5bd0b790' -ServerInstance BC -TenantSpecificProperties -Tenant default | Where-Object { $_.IsInstalled -eq $true }
+                            if ($ConstructBaseDETestInstalled) {
+                                Write-Host "    Create W1 Test Suite"
+                                Invoke-NAVCodeunit `
+                                    -ServerInstance BC `
+                                    -CompanyName $companyName `
+                                    -CodeunitId 50000 `

```

Comment:
Würde nicht eine ID in unserem Bereich hierfür mehr Sinn machen?

Reply:
Aktuell sind unsere Tests halt im PTE Bereich, aber kann man renummerieren, klar. Möglich ist das. Geht man auf jeden Fall potentiellen ID Konflikten aus dem Weg 👍

Reply:
OK, macht jetzt wenig Sinn explizit für die eine Codeunit, aber lass uns bitte im Team mal diskutieren, ob wir das nicht generell für die Tests machen sollen. Oder hatten wir einen expliziten Grund, das so zu machen, den ich vergessen habe?

Reply:
Habe die Test-AppIds nun renummeriert und mit dem commit 74b1910 auf die neue ID verwiesen

Reply:
danke